core/scripts/release/package-test.sh

```diff
@@ -20,7 +20,9 @@ cd /tmp/lighthouse-local-test
 
 npm init -y
 npm install "$LH_ROOT/lighthouse-$VERSION.tgz"
-npm install lighthouse-plugin-publisher-ads@next
+# --force to ignore potentially off peer dependency on Lighthouse itself. This just ignores whatever
+# Lighthouse version pubads declares, and defers to the version installed above.
+npm install lighthouse-plugin-publisher-ads@next --force
```

Comment:
This is a test of using the package, but no one is going to use `--force` if trying to run both. We should reduce deviation from real use if possible.

Maybe we could update them to use `"lighthouse": ">8.6.0"` as a peer and we can mostly trust it because we run them in our CI?

Reply:
alternatively we can put this on the 10.0 TODO list and fix it before release (presumably esm `lighthouse-plugin-publisher-ads` will be in `@latest` by then, too)

Reply:
added to #14375